src/Steam.Common/SteamId.cs

```diff
@@ -0,0 +1,492 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Steam
+{
+    /// <summary>
+    /// A globally unique identifier for Steam accounts, groups, lobbies, and chat rooms
+    /// </summary>
+    public readonly struct SteamId : IEquatable<SteamId>
```

Comment:
I don't think it is a bad thing to link to the Steam API for more information about this construct, if we want to do this we should keep it consistent in the whole project (Currently app, depot, and package do have this)

Reply:
The Steam API docs are those docs
![image](https://user-images.githubusercontent.com/10121870/58666073-ff1af300-82f7-11e9-8ce1-1651e2e32775.png)

Reply:
That's all there is that's documented on their site? If it is then that it is fine like this